EIPS/eip-1057-1.md

```diff
@@ -0,0 +1,457 @@
+# Test Vectors for EIP-1057 - ProgPow
```

Comment:
This file should go into `assets/eip-1057/<chose a filename>`.

Reply:
Done